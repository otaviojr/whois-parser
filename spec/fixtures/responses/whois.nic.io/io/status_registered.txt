Domain Name: REDIS.IO
Registry Domain ID: D503300000040423931-LRMS
Registrar WHOIS Server: whois.name.com
Registrar URL: www.name.com
Updated Date: 2019-05-06T17:24:58Z
Creation Date: 2010-05-28T22:09:44Z
Registry Expiry Date: 2020-05-28T22:09:44Z
Registrar Registration Expiration Date:
Registrar: Name.com LLC
Registrar IANA ID: 625
Registrar Abuse Contact Email:
Registrar Abuse Contact Phone:
Reseller:
Domain Status: clientTransferProhibited https://icann.org/epp#clientTransferProhibited
Registrant Organization: Domain Protection Services, Inc.
Registrant State/Province: CO
Registrant Country: US
Name Server: NS2KQZ.NAME.COM
Name Server: NS1DJS.NAME.COM
Name Server: NS4BHT.NAME.COM
Name Server: NS3CKL.NAME.COM
DNSSEC: unsigned

>>> Last update of WHOIS database: 2019-06-25T20:05:14Z <<<

For more information on Whois status codes, please visit https://icann.org/epp

Access to WHOIS information provided by Internet Computer Bureau Ltd. ("ICB") is provided to assist persons in determining the contents of a domain name registration record in the ICB registry database. The data in this record is provided by ICB for informational purposes only, and ICB does not guarantee its accuracy. This service is intended only for query-based access. You agree that you will use this data only for lawful purposes and that, under no circumstances will you use this data to(i) allow, enable, or otherwise support the transmission by e-mail, telephone, facsimile or other electronic means of mass, unsolicited, commercial advertising or solicitations to entities other than the data recipient's own existing customers; or (ii) enable high volume, automated, electronic processes that send queries or data to the systems of Registry Operator, a Registrar, or ICB or its services providers except as reasonably necessary to register domain names or modify existing registrations. UK privacy laws limit the scope of information permitted for certain public access.  Therefore, concerns regarding abusive use of domain registrations in the ICB registry should be directed to either (a) the Registrar of Record as indicated in the WHOIS output, or (b) the ICB anti-abuse department at abuse@icbregistry.info.

All rights reserved. ICB reserves the right to modify these terms at any time. By submitting this query, you agree to abide by these policies

The Registrar of Record identified in this output may have an RDDS service that can be queried for additional information on how to contact the Registrant, Admin, or Tech contact of the queried domain name.
