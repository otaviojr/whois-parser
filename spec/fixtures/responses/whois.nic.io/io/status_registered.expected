#disclaimer
  %s %ERROR{AttributeNotSupported}


#domain
  %s == "redis.io"

#domain_id
  %s == "D503300000040423931-LRMS"


#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2010-05-28 22:09:44 UTC}

#updated_on
  %s %CLASS{time}
  %s %TIME{2019-05-06 17:24:58 UTC}

#expires_on
  %s %CLASS{time}
  %s %TIME{2020-05-28 22:09:44 UTC}


#registrar
  %s %CLASS{registrar}
  %s.id           == "625"
  %s.name         == "Name.com LLC"
  %s.organization == "Name.com LLC"
  %s.url          == "www.name.com"

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].id            == nil
  %s[0].name          == nil
  %s[0].organization  == "Domain Protection Services, Inc."
  %s[0].address       == nil
  %s[0].city          == nil
  %s[0].zip           == nil
  %s[0].state         == "CO"
  %s[0].country       == nil
  %s[0].country_code  == "US"
  %s[0].phone         == nil
  %s[0].fax           == nil
  %s[0].email         == nil
  %s[0].created_on    == nil
  %s[0].updated_on    == nil

#admin_contacts
  %s %ERROR{AttributeNotSupported}

#technical_contacts
  %s %ERROR{AttributeNotSupported}


#nameservers
  %s %CLASS{array}
  %s %SIZE{4}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns2kqz.name.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns1djs.name.com"
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns4bht.name.com"
  %s[3] %CLASS{nameserver}
  %s[3].name == "ns3ckl.name.com"


#reserved?
  %s == false
